Makefile.mk

```diff
@@ -155,6 +155,15 @@ OBJ = $(OBJ_C) $(OBJ_S)
 DEP = $(OBJ:%.o=%.d)
 
 ################################################################################
+# Open project in Visual Code                                                  #
+################################################################################
+
+.PHONY: code
+code:
+	$(Q)$(ECHO) "Opening project in Visual Code..."
+	$(Q)code .
```

Comment:
Similar like OZONE variable, I suggest this to be specified in the beginning of file with ?= convention so it may get overridden from environmental variables, e.g. `$ VSCODE=<path> make code`